### DISCUSSION TITLE
Copy and pasting blocks

### DISCUSSION THREAD

**Discussion by Cweepaw** [chrislo27/PolyrhythmMania#10] [Ideas]
# Copy and Pasting blocks
Allow Ctrl-C and Ctrl-V (or buttons similar to the Playtest button) in the editor to place blocks identical to the copied block.

# Justification
Monotonous tasks such as pulsing every beat would be fixed by allowing copying and pasting. This will hopefully allow faster level creation.

# Potential Issues
There would be little to no way to tell where to put it, other than where the level maker's cursor would be. 

# Image Mockup
![mockup](https://user-images.githubusercontent.com/69701002/131169462-a9799732-d813-4a18-87d0-130b57523281.png)
![mockup2](https://user-images.githubusercontent.com/69701002/131169967-116b5af3-c97b-4ef8-9c56-07805172e1a9.png)

**Comment by chrislo27** [comment]
This is already implemented with alt+dragging. 

In the Editor Help menu (press F1) > Editor Controls and Keybinds > "When using the Selection tool". 
![image](https://user-images.githubusercontent.com/6299069/131173978-e243fdfd-67ae-462f-bd99-a3f2ee2de268.png)

**Comment by Cweepaw** [comment]
Ah, sorry.
